### ISSUE TITLE
Inconsistent types/completion/hover from submodules

### ISSUE THREAD

**Issue by jakebailey** [microsoft/pyright#433]
**Describe the bug**

In chains of submodules (like `import a.b.c`), the completions (members) and types of the modules are not consistent or complete.

**To Reproduce**

File structure like:

```
.
|-- app1.py
|-- app2.py
|-- projectA
|   `-- foo
|       |-- __init__.py
|       `-- bar
|           `-- __init__.py
`-- projectB
    `-- foo
        |-- __init__.py
        `-- baz
            `-- __init__.py
```

Init files empty; just markers.

Import things like `projectA.foo`, `projectA.foo.bar`, from imports.

**Expected behavior**

Completion, hovers, type info all work. Existence of members post import should work.

**Screenshots or Code**

`app1.py`:

```python
import projectA.foo
import projectA.foo.bar

# FAIL: Inconsistent completion, hover, etc.

projectA.

projectA.foo.
```

`app2.py`:

```python
import projectA.foo
from projectA.foo import bar

# FAIL: Inconsistent completion, hover, etc.

projectA.

projectA.foo.

bar.
```

The completion seems to be null; VS Code resorts to name completion:

![image](https://user-images.githubusercontent.com/5341706/70953082-1cf24000-201d-11ea-8db2-b7b8d90c40fa.png)

But, this one works:

![image](https://user-images.githubusercontent.com/5341706/70953151-4ad78480-201d-11ea-8b3f-d511d5e56a09.png)

Here it's `(module)`:

![image](https://user-images.githubusercontent.com/5341706/70953175-57f47380-201d-11ea-9787-61e5ac35977c.png)

But this one is `Module`:

![image](https://user-images.githubusercontent.com/5341706/70953209-65116280-201d-11ea-8325-943a9685a10d.png)

In `app2.py`, hovering over bar:

![image](https://user-images.githubusercontent.com/5341706/70953239-75294200-201d-11ea-98f3-0d5c7ce7f010.png)

**Comment by jakebailey** [comment]
Give me a few minutes; github decided to submit this issue before I had even finished pasting in my comments...

**Comment by jakebailey** [comment]
I have a slew of submodule-related issues to report (and I'm working to compress them down to more consumable examples). There's likely going to be some bug overlap so it's probably worth waiting to get the whole picture.

**Comment by erictraut** [comment]
Thanks for the bug report. These issues will be fixed in the next version of pyright.

**Comment by erictraut** [comment]
This is now fixed in version 1.1.12, which I just published.
